data/Makefile

```
@@ -772,17 +772,19 @@ min_path_size=200
 	mlr --tsvlite filter '$$n >= $n' $< >$@
 
 # Separate barcodes into molecules.
-mol_strategy=1
+mol_strategy="bc"
```

```suggestion
mol_strategy=bc
```
No need for quotes here.